### PR TITLE
TiKV: supplement missing release docs on optimizing the HA.

### PR DESCRIPTION
**Pull request by LykxSassinator**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

This pr is used to supplement extra missing release docs on optimizing the HA ability of TiKV.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by LykxSassinator** [comment]
This pr needs to cp to v7.5.1

File: releases/release-7.5.1.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ TiDB 版本：7.5.1
     - 允许多个快速加索引 DDL 任务排队执行，而非回退为普通加索引任务 [#47758](https://github.com/pingcap/tidb/issues/47758) @[tangenta](https://github.com/tangenta)
     - 提升空表加索引的速度 [#49682](https://github.com/pingcap/tidb/issues/49682) @[zimulala](https://github.com/zimulala)
 
++ TiKV
+
+    - 对慢节点检测的判定算法进行了改善，提升了算法检测的灵敏度，同时降低了算法对于热读写场景下的误判率。[#15909](https://github.com/tikv/tikv/issues/15909) @[LykxSassinator](https://github.com/LykxSassinator)
```
Comment (hfxsd) [inline releases/release-7.5.1.md:48]:
```suggestion
    - 改善慢节点检测的判定算法，提升算法检测的灵敏度，同时降低了算法对于热读写场景下的误判率 [#15909](https://github.com/tikv/tikv/issues/15909) @[LykxSassinator](https://github.com/LykxSassinator)
```

File: releases/release-7.5.1.md [hunk 2]
```diff
@@ -159,6 +163,7 @@ TiDB 版本：7.5.1
     - 修复扩容时可能导致 DR Auto-Sync 的 joint state 超时问题 [#15817](https://github.com/tikv/tikv/issues/15817) @[Connor1996](https://github.com/Connor1996)
     - 修复 Resolved TS 可能被阻塞两小时的问题 [#11847](https://github.com/tikv/tikv/issues/11847) [#15520](https://github.com/tikv/tikv/issues/15520) [#39130](https://github.com/pingcap/tidb/issues/39130) @[overvenus](https://github.com/overvenus)
     - 修复 `cast_duration_as_time` 可能返回错误结果的问题 [#16211](https://github.com/tikv/tikv/issues/16211) @[gengliqi](https://github.com/gengliqi)
+    - 修复 TiKV 在 Raftstore 阻塞在 I/O 操作上时异常假死的问题 [#16408](https://github.com/tikv/tikv/pull/16408) @[LykxSassinator](https://github.com/LykxSassinator)
```
Comment (hfxsd) [inline releases/release-7.5.1.md:166]:
```suggestion
    - 修复 TiKV 在某些异常场景下（如阻塞在 Disk I/O 操作上）假死而影响可用性的问题 [#16368](https://github.com/tikv/tikv/issues/16368) @[LykxSassinator](https://github.com/LykxSassinator)
```

File: releases/release-7.5.1.md [hunk 3]
```diff
@@ -159,6 +163,7 @@ TiDB 版本：7.5.1
     - 修复扩容时可能导致 DR Auto-Sync 的 joint state 超时问题 [#15817](https://github.com/tikv/tikv/issues/15817) @[Connor1996](https://github.com/Connor1996)
     - 修复 Resolved TS 可能被阻塞两小时的问题 [#11847](https://github.com/tikv/tikv/issues/11847) [#15520](https://github.com/tikv/tikv/issues/15520) [#39130](https://github.com/pingcap/tidb/issues/39130) @[overvenus](https://github.com/overvenus)
     - 修复 `cast_duration_as_time` 可能返回错误结果的问题 [#16211](https://github.com/tikv/tikv/issues/16211) @[gengliqi](https://github.com/gengliqi)
+    - 修复 TiKV 在某些异常场景下 (如阻塞在 Disk I/O 操作上) 假死而影响可用性的问题 [#16408](https://github.com/tikv/tikv/pull/16408) @[LykxSassinator](https://github.com/LykxSassinator)
```
Comment (hfxsd) [inline releases/release-7.5.1.md:166]:
```suggestion
    - 修复 TiKV 在某些异常场景下 (如阻塞在 Disk I/O 操作上) 假死而影响可用性的问题 [#16368](https://github.com/tikv/tikv/issues/16368) @[LykxSassinator](https://github.com/LykxSassinator)
```

File: releases/release-7.5.1.md [hunk 4]
```diff
@@ -159,6 +163,7 @@ TiDB 版本：7.5.1
     - 修复扩容时可能导致 DR Auto-Sync 的 joint state 超时问题 [#15817](https://github.com/tikv/tikv/issues/15817) @[Connor1996](https://github.com/Connor1996)
     - 修复 Resolved TS 可能被阻塞两小时的问题 [#11847](https://github.com/tikv/tikv/issues/11847) [#15520](https://github.com/tikv/tikv/issues/15520) [#39130](https://github.com/pingcap/tidb/issues/39130) @[overvenus](https://github.com/overvenus)
     - 修复 `cast_duration_as_time` 可能返回错误结果的问题 [#16211](https://github.com/tikv/tikv/issues/16211) @[gengliqi](https://github.com/gengliqi)
+    - 修复 TiKV 在某些异常场景下 (如阻塞在 Disk I/O 操作上) 假死而影响可用性的问题 [#16368](https://github.com/tikv/tikv/issues/16368) @[LykxSassinator](https://github.com/LykxSassinator)
```
Comment (qiancai) [inline releases/release-7.5.1.md:166]:
```suggestion
    - 修复 TiKV 在某些异常场景下（如阻塞在 Disk I/O 操作上）假死而影响可用性的问题 [#16368](https://github.com/tikv/tikv/issues/16368) @[LykxSassinator](https://github.com/LykxSassinator)
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**